### PR TITLE
feat: #2615 Migrate Alert, Info and Helper to Elements v2.0

### PR DESCRIPTION
**Pull request by trankhacvy**
# Pull request checklist

## Does this close any currently open issues?

fixes: https://github.com/reapit/foundations/issues/2615

**Please check if your PR fulfills the following requirements:**

- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been reviewed and added / updated if needed (for bug fixes / features)
- [x] Build (`yarn build`) was run locally and any changes were pushed
- [x] Lint (`yarn lint`) has passed locally and any fixes were made for failures
- [x] Test (`yarn test`) has passed locally and any fixes were made for failures

## Pull request type

**Please check the type of change your PR introduces:**

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

## What is the current behavior?

Issue Number: #

## What is the new behavior?

-

## Does this introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

File: packages/elements-next/src/components/message/message.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,39 @@
+import React from 'react'
+// also exported from '@storybook/react' if you can deal with breaking changes in 6.1
+import { Story, Meta } from '@storybook/react/types-6-0'
+
+import { Message, MessageProps } from './message'
+
+export default {
+  title: 'Example/Message',
```
Comment (tanphamhaiduong) [inline packages/elements-next/src/components/message/message.stories.tsx:8]: I think `Message` is better

File: packages/elements-next/src/components/message/message.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  component: Message,
+  argTypes: {
+    backgroundColor: { control: 'color' },
```
Comment (tanphamhaiduong) [inline packages/elements-next/src/components/message/message.stories.tsx:11]: This one will be props of the argTypes

File: packages/elements-next/src/components/message/message.tsx [hunk 3]
```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+import { cx } from 'linaria'
+import { elMessage } from './styles'
+import { elBgPrimary, elBgInfo, elBgSuccess, elBgDanger } from '@/base/background-colors'
+
+export interface MessageProps {
+  /**
+   * Message content
+   */
+  message?: string
```
Comment (nphivu414) [inline packages/elements-next/src/components/message/message.tsx:10]: message prop should be required for `Message` component

**Review by nphivu414 — CHANGES_REQUESTED** [review]

File: packages/elements-next/src/components/message/message.stories.tsx (hunk 2 above)
Comment (tanphamhaiduong) [reply packages/elements-next/src/components/message/message.stories.tsx]: I think you can remove `argsTypes`

**Pull request merged**